Building isn't working

Hiya.

First of all, loving the idea on this, looks great so far :) 

Problem I've got is I can't compile the .ts! 

I get the following error

C:\Projects\WebGL\pixi-multistyle-text-themoonrat>npm run build

> pixi-multistyle-text@0.4.0 build C:\Projects\WebGL\pixi-multistyle-text-themoonrat
> tsc; browserify index.js -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d

error TS5023: Unknown compiler option 's'.
error TS5023: Unknown compiler option 'o'.

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! pixi-multistyle-text@0.4.0 build: `tsc; browserify index.js -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d`
npm ERR! Exit status 1

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
   },
   "scripts": {
     "demo": "npm run build && open demo/index.html",
-    "build": "tsc; browserify index.js -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d",
+    "build": "tsc && browserify index.js -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d",
     "prepublish": "npm run build"
   }
 }

